### PR TITLE
Minor Bug fix, Nearby pokemon logs etc.

### PR DESCRIPTION
**Pull request by joychugh**
* Show nearby Pokemon in logs
* Formatting changes
* Speed is now in meter per second instead of meter per two seconds
* #FIXME should be fixed ?
* Catch all nearby pokemon (randomly waiting b/w each catch)
* Retry once if bag is full to attempt to clear inventory and catch again.

### CODE REVIEW

File: pgoapi/pgoapi.py [hunk 1]
```diff
@@ -273,53 +275,64 @@ def cleanup_inventory(self, inventory_items=None):
...
                         self.log.info("Releasing pokemon: %s IV: %s", self.pokemon_names[str(pokemon['pokemon_id'])], pokemonIVPercentage(pokemon))
-                        self.release_pokemon(pokemon_id = pokemon["id"])
+                        self.release_pokemon(pokemon_id=pokemon["id"])
 
         return self.call()
 
+    @staticmethod
+    def can_we_evolve_this(inventory_item, pokemon_id):
+        return "pokemon_family" in inventory_item['inventory_item_data'] and \
+               inventory_item['inventory_item_data']['pokemon_family']['family_id'] == POKEMON_EVOLTION_FAMILY[
```
Comment (ZohaibMKhan) [inline pgoapi/pgoapi.py]: spelling mistake?

Comment (joychugh) [inline pgoapi/pgoapi.py]: where?

**Comment by ZohaibMKhan** [comment]
"POKEMON_EVOLTION_FAMILY"

File: pgoapi/pgoapi.py (hunk 1 above)
Comment (joychugh) [inline pgoapi/pgoapi.py]: good one haha ill fix

**Comment by j-e-k** [comment]
Thanks!

**Pull request merged**